verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -1423,6 +1430,51 @@ TEST_F(VerilogLanguageServerSymbolTableTest, CheckReferenceUnknownSymbol) {
   ASSERT_EQ(response_b["result"].size(), 0);
 }
 
+TEST_F(VerilogLanguageServerSymbolTableTest, HoverOverSymbol) {
+  absl::string_view filelist_content = "mod.v\n";
+  static constexpr absl::string_view  //
+      module_content(
+          R"(module mod(
+    input clk,
+    input reg [31:0] a,
+    input reg [31:0] b,
+    output reg [31:0] sum);
+  always @(posedge clk) begin : addition
+    assign sum = a + b;
```

Comment:
Maybe add a comment here that this is roughly here where we're hovering for the reader of the code matching the line with the content (as it is zero-indexed, it might be surprising at first)

```systemverilog
...
   assign sum = a + b;  // hovering over this sum
...
```